Update ecowitt.markdown (#27320)
home-assistant/home-assistant.github.io: source/_integrations/ecowitt.markdown

2. Pick your station -> Menu Others -> DIY Upload Servers.
3. Hit next and select 'Customized'
4. Pick the protocol Ecowitt, and put in the ip/hostname of your Home Assistant server.

> 5. Path have to match! If using the Ecowitt App, remove the first forward slash from the API token ad the app will prepend one itself.

Reviewer: ```suggestion
5. Path has to match! If using the Ecowitt App, remove the first forward slash from the API token, as the app will prepend one.
```
